app/ts/components/AbbreviatedValue.tsx

```diff
@@ -1,16 +1,28 @@
-import { parseNumericTerm } from '../library/utilities.js'
-
 export const AbbreviatedValue = ({ floatValue }: { floatValue: number }) => {
-	const { coefficient, exponent } = parseNumericTerm(floatValue)
+	const prefixes = [
+		{ value: 1e9, symbol: 'G' },
+		{ value: 1e6, symbol: 'M' },
+		{ value: 1e3, symbol: 'k' },
+	];
+
+	for (const prefix of prefixes) {
+		if (floatValue >= prefix.value) {
+			return <>{toFixedLengthDigits(floatValue / prefix.value) + prefix.symbol}</>
+		}
+	}
 
-	if (exponent < 0) {
-		const leadingZeros = Math.abs(exponent) - 1
-		return <>0.<small>{'0'.repeat(leadingZeros)}</small>{coefficient}</>
+	// if value is a fraction of 1
+	if (floatValue && floatValue % 1 === floatValue) {
```

Comment:
if floatValue = 100.1, this prints wrong values?

I feel we should have test cases for this calculation and it should be separated from the preact code so we can run tests for it

Reply:
this condition are for floats between 0 and 1 such as 0.0001; takes care of rendering the zeros after the decimal point as smaller font size.